### PR TITLE
Update differences-classic-modern-search.md

### PR DESCRIPTION
**Pull request by Josardi**
Hi , Im Support Escalation Engineer for SharePoint Online at Microsoft, Recently i had a discussion with our PG due a customer was complaining about that Result sources were not working.

And PG said that once the tenant is fully in Microsoft Search , Custom result sources are not supported, and suggested use Microsoft Search Verticals

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/differences-classic-modern-search.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ The most visible difference is that the Microsoft Search box is placed at the to
 
 Search admin can customize the *classic* search experience, but not the Microsoft Search experience. As a search admin you can *tailor* Microsoft Search to your organization so it's easy for your users to find often needed content in your organization.
 
+For Example if your organization have Microsoft Search full deployed , Custom Result sources at site collection or tenant level won't affect the search result instead search admin can use Microsoft Search Verticarls [Learn about Manage search verticals](https://docs.microsoft.com/en-us/microsoftsearch/manage-verticals)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/differences-classic-modern-search.md:32]:
```suggestion
For example, if your organization has Microsoft Search full deployed, custom result sources at site collection or tenant level won't affect the search result. The search admin can use Microsoft search verticals instead. [Learn about Manage search verticals](https://docs.microsoft.com/en-us/microsoftsearch/manage-verticals).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Josardi, kindly commit the suggested changes. Many thanks.

**Review by Josardi — COMMENTED** [review]
Changes committed

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Josardi** [comment]
Hello @JohanFreelancer9  what are the next steps?

Thanks in advance

**Comment by scanum** [comment]
@SerdarSoysal 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**